### PR TITLE
fix height of history pane

### PR DESCRIPTION
**Pull request by chovy**
## Ticket
* https://issues.aphelion-neo.com/tktview/5751bde402af18134e1bffcd90075477d637a27c

## Changes
* use viewport height instead of percentage so resized window scales well.

## Screenshots
screenshot 2018-08-03 11 22 07

## Code Coverage

n/a

### CODE REVIEW

**Comment by chovy** [comment]
how would you specify a height so you trigger an overflow scrollbar using flex?

**Comment by crishellco** [comment]
@chovy a combination of `flex-direction: column` `flex: 1` `flex: none` `overflow: hidden` `overflow: auto`. Somewhere in the DOM tree a necessary rule is missing. I'd start there.

**Comment by chovy** [comment]
I'm not seeing a way to do this without using a height.

File: src/renderer/components/history/Table.vue [hunk 1]
```diff
@@ -192,8 +194,9 @@ export default {
   }
 
   .body {
-    height: 90%;
+    flex-grow: 1;
     overflow-y: auto;
+    min-height: 0; /* for Firefox */
```
Comment (crishellco) [inline src/renderer/components/history/Table.vue]: We don't need anything for Firefox. Electron uses Chrome.

Comment (chovy) [reply src/renderer/components/history/Table.vue]: I know, its just that if we ever used this code in firefox, we would need that and would probably never know to do it.

**Comment by crishellco** [comment]
@chovy Fixed it using flex without height.

**Comment by chovy** [comment]
Thanks.

**Pull request merged**